service/src/main/java/com/ge/predix/acs/config/GraphConfig.java

```diff
@@ -90,6 +94,17 @@ public Graph createGraph() throws InterruptedException {
             newGraph = TitanFactory.build().set("storage.backend", "inmemory").open();
         }
 
+        // TODO: Do we need a composite or mixed index for version vertex?
```

Comment:
Please capture this in a story. Its a nice to have, since we only make this query  once on startup.